ui/ui-components/pages/app/[id]/refresh.tsx

```diff
@@ -60,11 +64,11 @@ export default function Page(props) {
     );
 
     if (response?.appRefreshQuery) {
-      setAppRefreshQuery(response.appRefreshQuery);
-
+      setAppRefreshQuery({ ...response.appRefreshQuery });
```

Comment:
Not sure this spread is required.

Reply:
Ah, ty!  This was trying to troubleshoot the mysterious null problem.